src/scheduler/node_info.c

```diff
@@ -310,6 +339,13 @@ query_node_info(struct batch_status *node, server_info *sinfo)
 			if (*endp == '\0')
 				ninfo->port = count + 1;
 		}
+		else if(!strcmp(attrp->name, ATTR_partition)) {
+			ninfo->partition = string_dup(attrp->value);
+			if (ninfo->partition == NULL) {
+				log_err(errno, "node_queue_info", MEM_ERR_MSG);
```

Comment:
don't use log_err(), use schdlog().  Plus use \_\_func\_\_

Reply:
Have changed to use \__func__. But since it is system error I think it is better to use log_err() with errno. Any specific reason why only schdlog() is to be used in this case ?

Reply:
The scheduler uses schdlog().  If you don't use it, then log_filter will fail.  Please use schdlog().  Every malloc() failure in the scheduler uses the same message (MEM_ERR_MSG) with schdlog().

Reply:
Thanks for letting me know this. I have made the changes accordingly. It seems there are several places where still log_err() is used with MEM_ERR_MSG which lead to my initial confusion.

Reply:
There are?  There shouldn't be.  I'm sorry for the confusion.  Everything in the scheduler should use schdlog() until we move to a more uniform logging framework for all of PBS (or even move the scheduler to log_events).